src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/JobEndpoint.java

```diff
@@ -202,6 +205,15 @@ public void processJob(
 
     if (job.getJobStatus() == JobStatus.VALIDATED_OK
         || job.getJobStatus() == JobStatus.VALIDATED_WITH_ERRORS) {
+
+      if (job.getCollectionExercise()
+          .getSurvey()
+          .getSampleDefinitionUrl()
+          .endsWith("business.json")) {
```

Comment:
I appreciate that this is sort of hardcoded at the moment, grabbing the end of the URL etc.   Could this be done nicely?  Not asking for an elegant solution now, but a TODO comment would be nice.  Or/and stick it all in another well named function, remove some complexity from this already complex area.

Something like
private void setUpAndPostSampleToRasRMIfBusiness(Job job)

Reply:
I'd prefer something like a new type/setting on the survey, instead of string comparing